tests/functional/fx_fennec_v1_sign_up.js

```diff
@@ -58,6 +58,20 @@ define([
         })
 
         .then(FunctionalHelpers.testIsBrowserNotified(self, 'fxaccounts:can_link_account'))
```

Comment:
@shane-tomlinson oh ok, I guess I got confused with the refactor, I'm gonna go through it again to double check.